deps/exokit-bindings/videocontext/include/VideoCommon.h

```diff
@@ -0,0 +1,57 @@
+#ifndef _HTML_VIDEO_COMMON_ELEMENT_H_
+#define _HTML_VIDEO_COMMON_ELEMENT_H_
+
+#if defined(_WIN32)
+#define EXO_OS_WIN 1
+#elif defined(__APPLE__)
+#define EXO_OS_OSX 1
+#else
+#define EXO_OS_LINUX 1
+#define EXO_USING_V4L 1
+#endif
+
+#include <string>
+#include <vector>
+#include <sstream>
+
+namespace ffmpeg {
+
+  static int Abs(int a) {
```

Comment:
Could we not remplate this?

Reply:
@shawwn yeah I don't get the point of the type specifications when these could be type templates.